packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -774,6 +748,32 @@ class DropdownCore extends React.Component<Props, State> {
         );
     }
 
+    handleSearchTextChanged: (searchText: string) => void = (
+        searchText: string,
+    ) => {
+        const {onSearchTextChanged} = this.props;
+
+        if (onSearchTextChanged) {
+            onSearchTextChanged(searchText);
+        }
+    };
+
+    renderSearchField(): React.Node {
+        const {searchText} = this.props;
+        const {labels} = this.state;
+
+        return (
+            <SearchField
+                clearAriaLabel={labels.clearSearch}
+                onChange={this.handleSearchTextChanged}
+                placeholder={labels.filter}
+                ref={this.searchFieldRef}
+                style={styles.searchInputStyle}
+                value={searchText || ""}
+            />
+        );
+    }
+
```

Comment:
I moved "SearchTextInput" here by removing `search-text-input.js` altogether and simplifying this by calling the WB's `SearchField`s component directly.